docs/CONFIGURATION.md

```diff
@@ -242,9 +242,15 @@ The agent table configures Telegraf and the defaults used across all plugins.
 
 - **hostname**:
   Override default hostname, if empty use os.Hostname()
+
```

Comment:
I think they were deliberately together, they are also like that in config.go struct and in the sample config.